lib/rpm_database.rb

```diff
@@ -132,11 +133,15 @@ def parse_stat_line(line)
       "dir"
     when "symbolic link"
       "link"
-    when "regular file"
+    when / file$/
```

Comment:
I think I would match the style check by extracting the translation of the type output to the string we use in the JSON to a separate method.